src/usr/local/www/guiconfig.inc

```diff
@@ -492,15 +492,19 @@ function gentitle($title) {
 	}
 }
 
-function genhtmltitle($title) {
+function genhtmltitle($title, $links=array()) {
 
 	// If the array contains only one element, there are no breadcrumbs, so don't
 	// add anything else
 	if (count($title) > 1) {
 		$bc = '<ol class="breadcrumb">';
 
-		foreach ($title as $el) {
-			$bc .= '<li>'.$el.'</li>';
+		foreach ($title as $idx => $el) {
+			if (strlen($links[$idx]) > 0) {
+				$bc .= '<li><a href="https://' . $_SERVER['SERVER_ADDR'] . '/' . $links[$idx] . '">' .$el.'</a></li>';
```

Comment:
Do we need to manually add "https" or server name here. Currently When pages link to other pages, they mostly just use "PAGENAME.php for the target, and leave the server IP/URL, and the protocol (http/https) as it was. (also will this assumption of https break anything for non https usage?)